Makefile

```diff
@@ -140,6 +151,14 @@ $(TARGET).bin: $(TARGET).elf
 flash:
 	$(SF) write $(TARGET).bin 0x8000000
 
+ventilation_test: test/testRunner.c src/application/ventilation.o test/ventilation-test.o test/stubs/systemInfo_stub.o test/stubs/valve_stub.o test/stubs/dss_stub.o test/testUtil.o
```

Comment:
Please split into multiple lines using the `\` separator.

Reply:
done